tests/artifactorygroup_test.go

```diff
@@ -0,0 +1,113 @@
+package tests
+
+import (
+	"fmt"
+	artifactorynew "github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/artifactory/auth"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/jfrog/jfrog-client-go/config"
+	"github.com/stretchr/testify/assert"
+	"math/rand"
+	"testing"
+)
+
+func TestGroups(t *testing.T) {
+	t.Run("create", testCreateGroup)
+	t.Run("update", testUpdateGroup)
+	t.Run("delete", testDeleteGroup)
+}
+
+func testCreateGroup(t *testing.T) {
```

Comment:
Let's make sure that all tests which create a group, also delete it at the end.

Reply:
I can do that, but it then becomes one more thing that needs to succeed for the test to pass even though it's unrelated. Hence, why I create random. Also, I added the use of a docker container for testing so that the container just terminates at the end of the test run and everything is tossed.